youtube-disable-number-seek.js

```diff
@@ -6,6 +6,24 @@
  * https://github.com/ccampbell/mousetrap/blob/master/mousetrap.js
  */
 
+const settings = {
+    disable_0: true,
+    disable_home: true,
+    disable_end: true,
+};
+
+const keys = Object.keys(settings);
+
+browser.storage.local.get(keys).then(result => {
+    for (const key of keys) {
+        if (result[key] !== undefined) {
+            settings[key] = result[key];
+        } else {
+            browser.storage.local.set({[key]: true});
+        }
+    }
+});
```

Comment:
Dealing with asynchronous functions is new to me, does anything look like a problem here?